standalone/standalone.py

```diff
@@ -100,27 +116,45 @@
           containers:
             - args:
                 - |
-                  mkdir -p /output/model;
-                  mkdir -p /output/data;
-                  python3.11 -u run_main_ds.py --model_path {path_to_model} --ckpt_output_dir /output/model --data_output_dir /input_data/processed_data
+                  mkdir -p /data/model;
+                  mkdir -p /data/data;
+                  mkdir -p {path_to_model}/output
+                  export XDG_CACHE_HOME=/tmp
+                  export TRITON_CACHE_DIR=/tmp
+                  export HF_HOME=/tmp
+                  export TRANSFORMERS_CACHE=/tmp
+                  torchrun --nnodes {nnodes} \
```

Comment:
That doesn't seem fully equivalent to running the `run_main_ds.py`, e.g. for options validation. Maybe the logic from the `run_training` in `instructlab.training.main_ds` could be used?

Reply:
I'm deferring this to @MichaelClifford who originally implemented this code.